### PR TITLE
REF: Replacing `axis in` with `_get_axis_number(axis)`

### PR DESCRIPTION
**Pull request by RohanTrix**
xref [#43048](https://github.com/pandas-dev/pandas/issues/43048). Refactored test_label_or_level_utils.py

### CODE REVIEW

**Comment by attack68** [comment]
lgtm pending green tests

File: pandas/tests/generic/test_label_or_level_utils.py [hunk 1]
```diff
@@ -130,14 +133,16 @@ def test_is_level_reference_series_axis1_error(df):
...
         df_ambig = df_ambig.T
+        msg = "'L1' is both a column level and an index label"
 
-    if axis in {0, "index"}:
-        msg = "'L1' is both an index level and a column label"
     else:
-        msg = "'L1' is both a column level and an index label"
+        msg = "'L1' is both an index level and a column label"
+    
+
```
Comment (attack68) [inline pandas/tests/generic/test_label_or_level_utils.py:145]: remove these blank lines. You get CI check errors

Comment (RohanTrix) [reply pandas/tests/generic/test_label_or_level_utils.py]: Done

**Review by attack68 — APPROVED** [review]

**Comment by jreback** [comment]
thanks @RohanTrix very nice. Leavning the issue open if there are additional cases.

**Comment by RohanTrix** [comment]
Thanks @jreback !

**Pull request merged**